web5-test-vectors/vector-structure.json

```diff
@@ -0,0 +1,35 @@
+{
+    "description": "Describe the feature under test",
+    "numberOfTests": 3,
+    "vectors": [
+        {
+            "testId": 1,
+            "description": "describe this test case",
+            "input": "input string",
+            "output": {
+                "successful": "return object"
+            },
+            "result": "valid"
```

Comment:
What's the difference between `output` and `result`?

Reply:
what is the intended usage pattern for `result`?

Reply:
Discussed sync - will rename this to `errors` and use bool true/false to indicate if the feature being tested should return an error without any output